app/gui2/src/components/GraphEditor/widgets/WidgetFunction.vue

```diff
@@ -21,6 +26,69 @@ const application = computed(() => {
     info?.methodCall.notAppliedArguments ?? [],
   )
 })
+
+const selfArgumentAstId = computed<Opt<ExprId>>(() => {
+  const tree = props.input
+  if (tree.isTree(Tree.Type.OprApp)) {
+    return tree.tryMap((tree) => tree.lhs)?.astId
+  } else if (tree.isTree(Tree.Type.App)) {
+    return tree.tryMap((tree) => tree.arg)?.astId
+  } else if (tree.isTree(Tree.Type.NamedApp)) {
+    return tree.tryMap((tree) => tree.arg)?.astId
+  } else if (tree.isTree(Tree.Type.Ident)) {
+    return tree.astId
+  }
+  return null
+})
```

Comment:
Once this PR is rebased, this logic should be based on `InterpretedCall` type, to make both prefix and infix calls work consistently.

Reply:
Done